CHANGELOG.md

```diff
@@ -1,3 +1,72 @@
+# 0.0.118 - Oct XXX, 2023 - "Just the Twelve Sinks"
+
+## API Updates
+ * BOLT12 sending and receiving is now supported as an alpha feature. You may
+   run into unexpected issues and will need to have a direct connection with
+   onion message peer targets in order to exchange messages. We are seeking
```

Comment:
Maybe say "payment recipient" instead of "onion message peer targets in order to exchange messages"? Simpler and also includes the payment portion of the flow, which also requires a direct connection.

Reply:
I guess it is something like "payment recipient, which must also be the introduction node in blinded paths".

Reply:
#2540 is listed in #1970 for "Forwarding and receiving to multi-hop blinded paths"

Reply:
Yeah, that's right. I was confusing myself about support for blinded payment paths, actually. We can find a payment path to them, though, so it's "the offer's recipient or the introduction node of its blinded paths".